r/the-tidyfinance-r-package.qmd

```diff
@@ -0,0 +1,34 @@
+---
+title: The `tidyfinance` R package
+metadata:
+  pagetitle: The `tidyfinance` R package
+  description-meta: How to install and use the companion package to Tidy Finance with R.  
+---
+
+`tidyfinance` is an R package that contains a set of helper functions for empirical research in financial economics, addressing a variety of topics covered in this book. We designed the package to provide easy shortcuts for the applications that we discuss in the book. If you want to inspect the details of the package or propose new features, feel free to visit the package repository on [Github.](https://github.com/tidy-finance/r-tidyfinance)
+
+## Installation
+
+You can install the released version of `tidyfinance` [from CRAN](https://cran.r-project.org/package=tidyfinance) via:
+
+``` r
+install.packages("tidyfinance")
+```
+
+You can install the development version of `tidyfinance` from [GitHub](https://github.com/tidy-finance/r-tidyfinance) via, but keep in mind that it might not be fully tested:
+
+``` r
+# install.packages("pak")
+pak::pak("tidy-finance/r-tidyfinance")
+```
+
+## Usage
+
+Throughout the book, we refer to the corresponding features of the `tidyfinance` package. If you want to get an overview of the existing functionality, we suggest the blog posts that discuss specific releases and new features:
+
+- [Version 0.1.0](../blog/r-tidyfinance-0-1-0/index.qmd)
+- [Version 0.4.0](../blog/r-tidyfinance-0-4-0-new-data-downloads/index.qmd)
+
+## Feature requests
+
+We are curious to learn in which direction we should extend the package, so please consider opening an issue in the package [repository.](https://github.com/tidy-finance/r-tidyfinance) For instance, we could support more data sources, add more parameters to the family of functions for data downloads, or we could put more emphasis on the generality of portfolio assignment or other modeling functions. Moreover, if you discover a bug, we are very grateful if you raise them in the repository.
```

Comment:
raise them? Maybe " if you discover a bug, we would be very grateful if you report the issue in our [Github repository]().